extensions/binding/org.eclipse.smarthome.binding.triggertest/ESH-INF/thing/thing-types.xml

```diff
@@ -51,6 +51,7 @@
         <kind>trigger</kind>
         <label>Trigger 2</label>
         <description>Description of trigger 2</description>
+        <event/>
```

Comment:
So in any case, the channel is free to send a string with the event then? This would mean we cannot declare that the channel nevers sends any string (the "null" situation).